packages/livechat/package.json

```diff
@@ -38,8 +38,7 @@
 		"@storybook/theming": "~6.5.16",
 		"@types/crypto-js": "~4.1.1",
 		"@types/mini-css-extract-plugin": "~1.4.3",
-		"@types/terser-webpack-plugin": "~4.2.2",
-		"@types/webpack": "~4.41.33",
+		"@types/webpack": "^5.28.4",
```

Comment:
Why we are removing terser plugin dependency:

<img width="717" alt="image" src="https://github.com/RocketChat/Rocket.Chat/assets/17487063/4e14bd71-47a4-4591-9e45-a415b472c595">